stack-overflow/src/view/QuestionView.js

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import '../styles/questionStyle.css';
+
+const QuestionView = ({ questions, onCreate, onChange, onChangeQuestion, onFilterByTitle, onFilterByTags, searchFieldTitle, searchFieldTag, showQuestions, logOut, title, text, tags, allTags }) => (
+    <div>
+        <div className="container-question">
+            <div className={`card-question card-primary`}>
```

Comment:
Sometimes you are using curly braces for class names, sometimes not. Curly brackets make sense only when you have a JS expression in there and not a constant string.